_docs-sources/pipelines/security/using-pipelines.md

```diff
@@ -1,4 +1,4 @@
-# Using Pipelines
+# Using Pipelines in Production
 
 Gruntwork Pipelines is designed to be used with a PR based workflow.
 This means an approval on a PR is an approval to deploy infrastructure, making the configuration of repo settings and branch protection especially important.
```

Comment:
If the focus on this page is branch protection, that makes me realize we should be a little more affirmative here about the importance of branch protection.

```suggestion
Gruntwork Pipelines is designed to be used with a Pull-Request (PR) based workflow. That means an approval on a PR is an _approval to execute code that will have full access to your AWS account_.

Pipelines will only execute code merged to the `main` or `master` branch of your `infrastructure-live` repository. The best way to carefully review what gets merged there is to thoughtfully configure both your repository and branch protection settings.
```

As a side note, do we recommend any repo settings, or just branch protection?